### DISCUSSION TITLE
Welcome to ⨋ Math Discussions!

### DISCUSSION THREAD

**Discussion by FabianaCampanari** [Quantum-Software-Development/Math#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by anonymoushim** [comment]
hello

**Comment by Monstersssssssssss** [comment]
Hello, blogger. I have a question about Akaike that I'd like to ask you. Simply put, I previously had three equations fitting one parameter. Now, I want to add one more parameter and two more equations. After adding them, it becomes five equations fitting two parameters. Does this hypothesis hold? Is it mathematically valid?
